region/maridia/inner-pink.json

```diff
@@ -4895,33 +5012,68 @@
                   "requires": [
                     "Gravity",
                     "Morph",
+                    {"or": [
+                      "SpaceJump",
+                      "canCarefulJump",
+                      "h_canResetFallSpeed",
+                      "canSuitlessMaridia",
+                      "h_canHBJ",
+                      { "enemyDamage": { "enemy": "Cacatac", "hits": 1, "type": "spike" } }
+                    ]},
                     {"or":[
                       "SpaceJump",
                       "canWalljump",
                       "HiJump",
-                      "canCarefulJump",
+                      "h_canIBJ",
```

Comment:
does this IBJ need a bomb horizontally with it?

Reply:
This path jumps to blocks above the spikes and bomb jumps there.